x-pack/plugins/security_solution/public/management/pages/event_filters/service/api_client.ts

```diff
@@ -7,24 +7,47 @@
 
 import { ENDPOINT_EVENT_FILTERS_LIST_ID } from '@kbn/securitysolution-list-constants';
 import type { HttpStart } from '@kbn/core/public';
+import type {
+  CreateExceptionListItemSchema,
+  UpdateExceptionListItemSchema,
+} from '@kbn/securitysolution-io-ts-list-types';
+import { removeIdFromExceptionItemsEntries } from '@kbn/securitysolution-list-hooks';
 import type { EndpointSuggestionsBody } from '../../../../../common/endpoint/schema/suggestions';
 import { SUGGESTIONS_ROUTE } from '../../../../../common/endpoint/constants';
 import { resolvePathVariables } from '../../../../common/utils/resolve_path_variables';
 import { ExceptionsListApiClient } from '../../../services/exceptions_list/exceptions_list_api_client';
 import { EVENT_FILTER_LIST_DEFINITION } from '../constants';
 
+function writeTransform<T extends CreateExceptionListItemSchema | UpdateExceptionListItemSchema>(
+  item: T
+): T {
+  return removeIdFromExceptionItemsEntries(item) as T;
+}
+
 /**
  * Event filters Api client class using ExceptionsListApiClient as base class
  * It follow the Singleton pattern.
  * Please, use the getInstance method instead of creating a new instance when using this implementation.
  */
 export class EventFiltersApiClient extends ExceptionsListApiClient {
   constructor(http: HttpStart) {
-    super(http, ENDPOINT_EVENT_FILTERS_LIST_ID, EVENT_FILTER_LIST_DEFINITION);
+    super(
+      http,
+      ENDPOINT_EVENT_FILTERS_LIST_ID,
+      EVENT_FILTER_LIST_DEFINITION,
+      undefined,
```

Comment:
Just looking from outside the context - wouldn't this be more convenient to pass the config as an object? Then you wouldn't have to pass undefineds just because the order defines it.

Curious of what you think about it, thanks!